packages/sdk-cli/src/models/LogConsumer.ts

```diff
@@ -77,7 +77,7 @@ export default class LogConsumer {
     }
 
     const sourceMapConsumers = await mapValues(lodash(sourceMaps).pickBy().value(), async maps =>
-      mapValues(maps!, async map => new SourceMapConsumer(map)));
+      mapValues(maps!, async map => new SourceMapConsumer(map as any)));
```

Comment:
What's the type-mismatch being avoided here?

Reply:
`map` is of type `unknown` so it needs to be cast to some other type to be used.
https://github.com/Fitbit/developer-bridge/blob/02139fcb46c640c98a6e363ae1c9d8d1a30626e1/packages/app-package/src/AppPackage.ts#L5-L9
An explicit cast is a big red flag that there is a possibly-unsafe assumption being made, which is exactly what's happening here. And you've proved that by noticing it.